index.html

```diff
@@ -4,11 +4,54 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>LAMBDA</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>LAMBDA - healthy food restaurant</title>
 </head>
+
 <body>
-    <h1>LAMBDA restaurant</h1>
-    
+
+    <header class="header">
+      <img class="logo" alt="LAMBDA" src="gallery/logo/lambda-logo.png">
+        <nav>
+          <ul class="main-nav-list">
+            <li><a class="main-nav-link" href="#">Home</a></li>
+            <li><a class="main-nav-link" href="#">About</a></li>
+            <li><a class="main-nav-link" href="#">Igredients</a></li>
+            <li><a class="main-nav-link" href="#">Menu</a></li>
+            <li><a class="main-nav-link" href="#">Reviews</a></li>
+            <li><a class="main-nav-link" href="#">Reservations</a></li>
+          </ul>
+            <ul class="social-links">
+              <li><a class="social-link" href="#"><img src="gallery/icon/twitter.png" name="logo-twitter"></a></li>
+              <li><a class="social-link" href="#"><img src="gallery/icon/youtube.png" name="logo-youtube"></a></li>
+              <li><a class="social-link" href="#"><img src="gallery/icon/facebook.png" name="logo-facebook"></a></li>            
+            </ul>
+        </nav>          
+    </header>
+
+      <main>
+
+        <section class="promo-section">
+          <div class="promo-banner">
+            <div class="promo-slogan"> 
+              <p class="slogan">the right ingredients<br>for the right food</p>
+                <div class="promo-decoration">
+                  <img src="gallery/decoration/decoration-1.png" class="decoration" alt="decoration-promo">
+                </div>
+                  <div class="buttons-bar">
+                    <div class="buttons">
+                      <a href="#" class="btn btn--full">BOOK A TABLE</a>
```

Comment:
Let's use CSS to make text uppercase.

Reply:
Edited. Added the necessary CSS properties